lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_oman.govspeak.erb

```diff
@@ -41,7 +41,6 @@
   <%= render partial: 'consular_fees_table.govspeak.erb',
              locals: {
                services: [
-                 :administering_oath_or_making_declaration,
```

Comment:
See comments and screenshots [here](https://github.com/alphagov/smart-answers/pull/2413/files#r57885971)